docs/audits.md

```diff
@@ -693,6 +693,52 @@ like `pull_request_target`.
 
 If you need to pass state between steps, consider using `GITHUB_OUTPUT` instead.
 
+## `cache-poisoning`
+
+| Type     | Examples                | Introduced in  | Works offline  | Enabled by default |
+|----------|-------------------------|----------------|----------------|--------------------|
+| Workflow  | [cache-poisoning.yml]   | v1.0.0         | ✅             | ✅                 |
```

Comment:
Went for `1.0.0` since you added the milestone 😄

Reply:
Good call, although I _think_ I'm going to do at least one more minor release before going to 1.0 😅 -- so let's do this as `v0.10.0`, which I'll plan to have out pretty soon:

```suggestion
| Workflow  | [cache-poisoning.yml]   | v0.10.0         | ✅             | ✅                 |
```